dfinity/nns-dapp
# GIX-2060: Remove transactionFeesStore

# Motivation

Clean up unused code. We have migrated all the readings of the token fee to use the `tokensStore`. Therefore, the `transactionsFeesStore` is not used anymore.

In this PR, remove the `transactionsFeesStore` and all its references.

# Changes

* Remove the store from the debug store.
* Remove the code that populated the transactionsFeesStore in loadSnsProjects.
* Remove calling `loadSnsTransactionFee` on syncSnsAccounts.
* Remove the transaction-fees services file. It has only one service: `loadSnsTransactionFee`.
* Remove the transaction-fees store file.

# Tests

* Remove test files for transaction-fees services and stores.
* Remove the check on transactionsFeesStore in loadSnsProjects test.
* Remove resets and sets of transactionsFeesStore in various tests. This was legacy and it shows that the tests pass without any further change.

# Todos
